src/System.Reflection.Extensions/ref/4.0.0/System.Reflection.Extensions.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <OutputType>Library</OutputType>
+    <PackageTargetFramework>netstandard1.0</PackageTargetFramework>
```

Comment:
>netstandard1.0 [](start = 28, length = 14)

This should be netstandard1.3, NuGetTargetMoniker the same, and same for the project.json